scripts/amd/gemm/matmul.py

```diff
@@ -218,7 +219,7 @@ def matmul(a, b, activation=""):
     M, K = a.shape
     K, N = b.shape
     # Allocates output.
-    c = torch.empty((M, N), device=a.device, dtype=a.dtype)
+    c = torch.zeros((M, N), device=a.device, dtype=torch.float32)
```

Comment:
We cannot always use fp32 since this will affect performance. so this line should be change to:
`c = torch.zeros((M, N), device=a.device, dtype=a.dtype)`

Reply:
done!